src/main/java/ui/windows/StatsDisplay.java

```diff
@@ -63,35 +50,35 @@ private void initializeGUIComponents() {
 
         // Link the search bar to the delegator
         inputNameField.addActionListener(delegator);
-
+        
         // Formatting/spacing TBD
         buttonPanel.add(backButton);
         buttonPanel.add(Box.createHorizontalGlue());
         buttonPanel.add(inputNameField);
 
-        setLayout(new BorderLayout());
-        add(dataPanel, BorderLayout.CENTER);
-        add(buttonPanel, BorderLayout.PAGE_END);
+        setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
+        add(dataPanel);
+        add(Box.createVerticalGlue());
+        add(buttonPanel);
     }
 
     /**
      * Update this instance's labels to reflect updated values in data.
      */
     public void updateView(StatsDisplayData data) {
-        nameLabel.setText("User: " + data.getName());
-        gamesPlayedLabel.setText("Games Played: " + data.getGamesPlayed());
+        nameLabel.setText(data.getName());
         gamesWonLabel.setText("Games Won: " + data.getGamesWon());
-        longestWinStreakLabel.setText("Longest Win Streak: " + data.getLongestWinStreak());
-        nextPlayer.setText("<html><br>Enter a new username to see statistics.</html>");
+        gamesLostLabel.setText("Games Lost: " + data.getGamesLost());
     }
 
     /**
      * Tests creation of a Stats window.
      */
     public static void main(String[] args) {
```

Comment:
I'd advise that once everything works to remove this as there should only be one main method in the entirety of the program.